vmcnet/utils/log_linear_exp.py

```diff
@@ -4,7 +4,8 @@
 import jax.numpy as jnp
 
 from vmcnet.utils.typing import Array, SLArray
-from vmcnet.utils.kfac import register_batch_dense
+
```

Comment:
@ggoldsh do u want to remove this?

Reply:
Oh good point, I left a lot of outdated references about registering kfac. Let me remove those since the new kfac auto detects how to register things.